mvlearn/decomposition/permica.py

```diff
@@ -0,0 +1,138 @@
+from .baseica import BaseICA
+from multiviewica import permica
+
+
+class PermICA(BaseICA):
```

Comment:
I'm not sure whether this algorithm should be at all in mvlearn. It was just a toy algorithm in our paper, and has never really been studied otherwise. Plus we don't really need it to initialize multiviewICA.

Reply:
Ok Let's remove it